src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -367,6 +367,7 @@ class SpfxProjectUpgradeCommand extends BaseProjectCommand {
         this.writeReport(findingsToReport, cmd, args.options);
         break;
       case 'tour':
+      case 'codetour':
```

Comment:
Is there a reason why should introduce two names for the same report type?

Reply:
I had a couple people reach out asking why tour vs CodeTour. I think tour is shorter and more consistent, so I'm not married to having a both tour and CodeTour.

Reply:
We also use `md` rather than `markdown`. I'd say let's stick to just `tour` for now 🙂

Reply:
Do you need me to resubmit without the added line?

Reply:
Nah, I can adjust it when merging the PR. All good 👍

Reply:
Removed `codetour` case